torchvision/prototype/datapoints/_datapoint.py

```diff
@@ -10,16 +10,16 @@
 from torchvision.transforms import InterpolationMode
 
 
-F = TypeVar("F", bound="_Feature")
+D = TypeVar("D", bound="_Datapoint")
 FillType = Union[int, float, Sequence[int], Sequence[float], None]
 FillTypeJIT = Union[int, float, List[float], None]
 
 
 def is_simple_tensor(inpt: Any) -> bool:
-    return isinstance(inpt, torch.Tensor) and not isinstance(inpt, _Feature)
+    return isinstance(inpt, torch.Tensor) and not isinstance(inpt, _Datapoint)
 
 
-class _Feature(torch.Tensor):
+class _Datapoint(torch.Tensor):
```

Comment:
Given that `_datapoint.py` is not public, we could rename to `Datapoint` and avoid exposing it via `__init__.py`. This way the internal and external teams looking at experimenting with this feature, won't have to rename everything on their side once we make it public. WDYT? No strong opinion on my side.

Reply:
I'm ok with that.